Orange/widgets/model/owknn.py

```diff
@@ -0,0 +1,65 @@
+from AnyQt.QtCore import Qt
+
+from Orange.data import Table
+from Orange.modelling import KNNLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+
+
+class OWKNNLearner(OWBaseLearner):
+    name = "K-Nearest Neighbors"
```

Comment:
It's a _new_, unified widget. I have no qualms about refreshing it. :stuck_out_tongue: